components/infobox/commons/infobox_person.lua

```diff
@@ -51,19 +51,33 @@ local _COUNTRIES_EASTERN_NAME_ORDER = {
 	'South Korea',
 	'Cambodia'
 }
-local STATUS_INACTIVE = 'inactive'
-local STATUS_BANNED = 'banned'
-local STATUS_RETIRED = 'retired'
+---@enum PlayerStatus
+local Status = {
+	ACTIVE = 'Active',
+	INACTIVE = 'Inactive',
+	RETIRED = 'Retired',
+	DECEASED= 'Deceased',
+}
+Status['PASSED AWAY'] = Status.DECEASED
+local BANNED = 'banned'
```

Comment:
What is the reason this is not in the enum?

Reply:
because as per my dms with rath it shouldn't be a status

![Screenshot 2023-12-13 101454](https://github.com/Liquipedia/Lua-Modules/assets/75081997/a501457a-2e5a-4183-a1d0-ee9554cd0db8)
![Screenshot 2023-12-13 101519](https://github.com/Liquipedia/Lua-Modules/assets/75081997/40e47ff5-f282-4fe8-be4b-973b5806c1c0)
![Screenshot 2023-12-13 101623](https://github.com/Liquipedia/Lua-Modules/assets/75081997/8d4dbbf2-642d-4ade-9ff0-a1c2eb528546)